From 478bdab5b7e6e4f479471e8df283d149c8a7d267 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.com>
Date: Thu, 12 Dec 2024 17:53:08 +0000
Subject: [PATCH] configs: Enable the WS2812_PIO_RP1 driver

Enable the WS2812_PIO_RP1 driver in both the defconfig files that
support the Pi 5 family.

Signed-off-by: Phil Elwell <phil@raspberrypi.com>
---
 arch/arm64/configs/bcm2711_defconfig | 1 +
 arch/arm64/configs/bcm2712_defconfig | 1 +
 2 files changed, 2 insertions(+)

--- a/arch/arm64/configs/bcm2711_defconfig
+++ b/arch/arm64/configs/bcm2711_defconfig
@@ -484,6 +484,7 @@ CONFIG_BLK_DEV_RBD=m
 CONFIG_BLK_DEV_NVME=y
 CONFIG_NVME_HWMON=y
 CONFIG_RP1_PIO=m
+CONFIG_WS2812_PIO_RP1=m
 CONFIG_SRAM=y
 CONFIG_EEPROM_AT24=m
 CONFIG_EEPROM_AT25=m
--- a/arch/arm64/configs/bcm2712_defconfig
+++ b/arch/arm64/configs/bcm2712_defconfig
@@ -487,6 +487,7 @@ CONFIG_BLK_DEV_RBD=m
 CONFIG_BLK_DEV_NVME=y
 CONFIG_NVME_HWMON=y
 CONFIG_RP1_PIO=m
+CONFIG_WS2812_PIO_RP1=m
 CONFIG_SRAM=y
 CONFIG_EEPROM_AT24=m
 CONFIG_EEPROM_AT25=m
